process/PC_PL_Template_Markdown.md

```diff
@@ -1,18 +1,30 @@
-# OpenHW Project Concept And Project Launch Markdown Template: Instructions
-*This template is divided into two parts:*
-- *The **Project Concept (PC)** required fields are shown in the first part.*
-- *The additional **Project Launch (PL)** required and optional fields are shown in the second part.*
-
-*Delete any sections not needed for your proposal*
-
-*The normal proposal flow is:*
-- ***The PC proposal*** *is prepared and presented to TWG. The PC proposal introduces the project and explains the market drivers and the "why"* 
-  - *TWG grants PC gate with feedback, or rejects PC gate with feedback*
-  - *If PC granted, additional work is carried out to prepare the **PL proposal***. 
-- *The **PL proposal** contains updates to the PC fields and adds additional fields. The PL proposal explains the "what" of the project.* 
-  - *For software porting projects, the PL should contain the feature list*
-  - *For IP core or other complex projects, the PL should contain a high level feature list (the user manual with feature specification is developed for the Plan Approved gate).*
-
+# OpenHW Project Concept and Project Launch markdown template: Instructions
+*This template presents both PC and PL templates as well as instructions for use*
+
+
+*In the **normal OpenHW project flow**:*
+- *The **PC*** 
+  - *Introduces the intitial project concept and team, associated market drivers or user requirements (the "why") along with basic scope of the project*
+  - *Typically lacks detail such as a feature list, architecture or complete project team.* 
+  - *If PC approved by the TWG, the project team completes the project description in the **PL proposal***. 
+  - *The PC as approved is retained in the programs repository*
+
+- *The **PL***:
+  - *Is a separate document to the PC, which it updates and completes.*
+  - *The architecture, major features and project phases (the "what" - but not necessarily final level of detail)*
```

Comment:
Maybe we can break this out to influence the authors of the PL to focus primarily on the features and project phases.
- *The major features of the IP (the "what")*
- *Top-level project phases of the IP development*
- *High level architecture of the IP (not necessarily final level of detail)*